lib/presentation/view/home.dart

```diff
@@ -1,15 +1,237 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:get/get.dart';
+
+import '../../core/util/api_constants.dart';
+import '../../core/util/asset_constants.dart';
+import '../../core/util/dimensions.dart';
+import '../../core/util/keys.dart';
+import '../../core/util/palette.dart';
+import '../../core/util/text_constants.dart';
+import '../controller/recipes_controller.dart';
+import '../widget/company_logo.dart';
+import '../widget/footer.dart';
+import '../widget/recipe_card.dart';
+import '../widget/recipe_card_vertical.dart';
 
 class Home extends StatelessWidget {
-  const Home({Key? key}) : super(key: key);
+  final RecipesController recipesController = Get.find<RecipesController>();
+
+  Home({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(
-        child: Text(
-          'Home placeholder',
-        ),
+    return Scaffold(
+      backgroundColor: Palette.mainBackground,
+      body: CustomScrollView(
+        shrinkWrap: true,
+        physics: const BouncingScrollPhysics(),
+        slivers: [
+          SliverLayoutBuilder(
+            builder: (BuildContext context, SliverConstraints constraints) {
+              return SliverAppBar(
+                shadowColor: Colors.black54,
+                pinned: true,
+                backgroundColor: Palette.primary,
+                actions: [
+                  Padding(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: MediaQuery.of(context).size.width *
+                          Dimensions.fivePercent,
+                    ),
+                    child: Image.asset(
+                      AssetConstants.searchIcon,
+                      width: Dimensions.searchIconWidth,
+                      color: Colors.white,
+                    ),
+                  ),
+                ],
+                title: Padding(
+                  padding: EdgeInsets.symmetric(
+                    horizontal: MediaQuery.of(context).size.width *
```

Comment:
You could replace MediaQuery.of(context).size.width to Get.width

Reply:
Done